docs/how_to/deploy_android.md

```diff
@@ -0,0 +1,85 @@
+How to deploy and use compiled model on Android
+===============================================
+
+This tutorial explain below aspects (Unlike the android_rpc approach we already have)
+
+    1: Build a model for android target.
+    2: TVM runtime building for android target.
+    3: A sample native application to test the model on ADB before integrating into android application (apk).
```

Comment:
I think we should first refer to the apps/deploy_android on how to use java API, the native API comes second